cmd/ssh.go

```diff
@@ -20,59 +20,65 @@ import (
 )
 
 const sshDescription = `
-For any nodes created through Rancher using docker-machine, 
+For any nodes created through Rancher using docker-machine,
 you can SSH into the node. This is not supported for any custom nodes.
 
 Examples:
 	# SSH into a node by ID/name
 	$ rancher ssh nodeFoo
 
-	# SSH into a node by name but specify the username to use
-	$ rancher ssh -l user1 nodeFoo
+	# SSH into a node by ID/name using the external IP address
+	$ rancher ssh -e nodeFoo
 
-	# SSH into a node by specifying user and node using the @ syntax while adding a command to run
-	$ rancher ssh user1@nodeFoo env
+	# SSH into a node by name but specify the login name to use
+	$ rancher ssh -l login1 nodeFoo
+
+	# SSH into a node by specifying login name and node using the @ syntax while adding a command to run
+	$ rancher ssh login1@nodeFoo env
```

Comment:
Update this example to include `--` for example `$ rancher ssh login1@nodeFoo -- netstat -p tcp`. The `--` is needed or flag parsing will drop flags from the command being passed through (`-p`) in this case.